README.md

```diff
@@ -15,254 +16,183 @@
-
-## About The Project
-
-Celo Composer allows you to quickly build, deploy, and iterate on decentralized applications using Celo. It provides a number of frameworks, templates, deployment and component support, and Celo specific functionality to help you get started with your next dApp. 
-
-It is the perfect lightweight starter-kit for any hackathon and for quickly testing out integrations and deployments on Celo.
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-## Built With
-
-Celo Composer is built on Celo to make it simple to build dApps using a variety of front-end frameworks, and libraries.
-
-- [Celo](https://celo.org/)
-- [Solidity](https://docs.soliditylang.org/en/v0.8.19/)
-- [Hardhat](https://hardhat.org/)
-- [React.js](https://reactjs.org/)
-- [Next.js](https://nextjs.org/)
-- [viem](https://viem.sh/)
-- [Tailwind](https://tailwindcss.com/)
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
-## Prerequisites
-
-- Node (v20 or higher)
-- Git (v2.38 or higher)
+---
 
-## How to use Celo Composer
+## **Table of Contents**
+1. [About The Project](#about-the-project)
+   - [Built With](#built-with)
+   - [Prerequisites](#prerequisites)
+2. [How to use Celo Composer](#how-to-use-celo-composer)
+   - [Install Dependencies](#install-dependencies)
+   - [Deploy a Smart Contract](#deploy-a-smart-contract)
+   - [Deploy Your Dapp Locally](#deploy-your-dapp-locally)
+   - [Add UI Components](#add-ui-components)
+   - [Deploy with Vercel](#deploy-with-vercel)
+   - [Supported Frameworks](#supported-frameworks)
+   - [Supported Templates](#supported-templates)
+3. [Usage](#usage)
+4. [Support](#support)
+5. [Roadmap](#roadmap)
+6. [Contributing](#contributing)
+7. [License](#license)
+8. [Contact](#contact)
 
-The easiest way to start with Celo Composer is using `@celo/celo-composer`. This CLI tool lets you quickly start building dApps on Celo, including several templates. To get started, just run the following command, and follow the steps:
-
-- Step 1
-
-```bash
-npx @celo/celo-composer@latest create
-```
+---
 
-- Step 2: Provide the Project Name: You will be prompted to enter the name of your project.
+## **About The Project**
 
-```text
-What is your project name: 
-```
+Celo Composer allows you to quickly build, deploy, and iterate on decentralized applications using Celo. It provides a variety of frameworks, templates, deployment tools, UI components, and Celo-specific functionality to accelerate your dApp development.  
 
-- Step 3: Choose to Use Hardhat: You will be asked if you want to use Hardhat. Select Yes or No.
+This is an ideal lightweight starter-kit for hackathons and rapid prototyping on Celo.
 
-```text
-Do you want to use Hardhat? (Y/n)
-```
+---
 
-- Step 4: Choose to Use a Template: You will be asked if you want to use a template. Select `Yes` or `No`.
+## **Built With**
+Celo Composer is built on the Celo ecosystem and supports multiple frameworks and libraries:
```

Comment:
"Celo Composer is built on Celo and supports multiple frameworks and libraries:"